tool/src/org/antlr/v4/tool/ErrorType.java

```diff
@@ -956,6 +956,8 @@
 	 */
 	CHANNELS_BLOCK_IN_COMBINED_GRAMMAR(164, "custom channels are not supported in combined grammars", ErrorSeverity.ERROR),
 
+	NONCONFORMING_LR_RULE(165, "rule <arg> is left recursive but doesn't conform to a pattern ANTLR can handle", ErrorSeverity.ERROR),
```

Comment:
oh sure.  it would be nice to renumber everything but it would  break all of the error numbers if people rely on them; I'm sure no one is relying on them but we might as well keep backward compatibility here. ok, so 169 it is.